pkg/helm/actions/testdata/chartmuseumWithBasicAuth.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+GOOS=${GOOS:-$(go env GOOS)}
+GOARCH=${GOARCH:-$(go env GOARCH)}
+./$GOOS-$GOARCH/chartmuseum --debug --port=8181 \
+  --storage="local" \
+  --storage-local-rootdir="./chartstore-8181" \
+  --basic-auth-user="AzureDiamond" \
```

Comment:
is this a real account ?

Reply:
This is `chartmuseum`'s authentication credentials for the test; please see https://knowyourmeme.com/memes/hunter2 for more information regarding the username/password pair :)